code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -48,57 +48,47 @@
 	if(!istype(C)) //Sanity doesn't hurt, right ?
 		return FALSE
 
 	if(ishuman(C))
-		var/obj/effect/equip_e/human/O = new /obj/effect/equip_e/human()
-		O.source = user
-		O.target = C
-		O.t_loc = C.loc
-		O.item = user.get_active_hand()
-		O.s_loc = user.loc
-		O.place = "handcuff"
-		C.requests += O
-		spawn()
-			if(istype(src, /obj/item/weapon/handcuffs/cable))
-				feedback_add_details("handcuffs", "C")
-			else
-				feedback_add_details("handcuffs", "H")
-			playsound(get_turf(src), cuffing_sound, 30, 1, -2)
-			O.process()
+		var/mob/living/carbon/human/H = C
+		if (!H.has_organ_for_slot(slot_handcuffed))
+			to_chat(user, "<span class='danger'>\The [C] needs at least two wrists before you can cuff them together!</span>")
+			return
 
-	else
-		var/obj/effect/equip_e/monkey/O = new /obj/effect/equip_e/monkey()
-		O.source = user
-		O.target = C
-		O.item = user.get_active_hand()
-		O.s_loc = user.loc
-		O.t_loc = C.loc
-		O.place = "handcuff"
-		C.requests += O
-		spawn()
-			playsound(get_turf(src), cuffing_sound, 30, 1, -2)
-			O.process()
+	playsound(get_turf(src), cuffing_sound, 30, 1, -2)
+	user.visible_message("<span class='danger'>[user] is trying to handcuff \the [C]!</span>",
+						 "<span class='danger'>You try to handcuff \the [C]!</span>")
 
-/obj/item/weapon/handcuffs/proc/handcuffs_remove(var/mob/living/carbon/C)
-	C.handcuffed = null
-	C.update_inv_handcuffed()
+	if(do_after(user, C, 3 SECONDS))
+		if(istype(src, /obj/item/weapon/handcuffs/cable))
+			feedback_add_details("handcuffs", "C")
+		else
+			feedback_add_details("handcuffs", "H")
 
-/obj/item/weapon/handcuffs/cyborg/attack(var/mob/living/carbon/M, var/mob/living/user, var/def_zone)
-	if (!istype(M) || M.handcuffed)
-		return FALSE
+		user.visible_message("<span class='danger'>\The [user] has put \the [src] on \the [C]!</span>")
+		user.attack_log += text("\[[time_stamp()]\] <font color='red'>Has put \the [src] on [C.name] ([C.ckey])</font>")
+		C.attack_log += text("\[[time_stamp()]\] <font color='red'>Handcuffed with \the [src] by [user.name] ([user.ckey])</font>")
+		log_attack("[user.name] ([user.ckey]) has cuffed [C.name] ([C.ckey]) with \the [src]")
 
-	playsound(get_turf(src), cuffing_sound, 30, 1, -2)
-	user.visible_message("<span class='danger'>[user] is trying to handcuff \the [M]!</span>",
-						 "<span class='danger'>You try to handcuff \the [M]!</span>")
-	if(do_after(user, M, 30))
-		M.handcuffed = new/obj/item/weapon/handcuffs(M)
-		M.update_inv_handcuffed()
+		var/obj/item/weapon/handcuffs/cuffs = src
+		if(istype(src, /obj/item/weapon/handcuffs/cyborg)) //There's GOT to be a better way to check for this.
+			cuffs = new(get_turf(user))
+		else
+			user.drop_from_inventory(cuffs)
+		cuffs.loc = C
+		C.handcuffed = cuffs
+		C.update_inv_handcuffed()
+
+/obj/item/weapon/handcuffs/cyborg
+//This space intentionally left blank
 
-	return TRUE
+/obj/item/weapon/handcuffs/proc/handcuffs_remove(var/mob/living/carbon/C)
+	C.handcuffed = null
```

Comment:
https://github.com/d3athrow/vgstation13/commit/e1050fd4bbc235b8dbf94d07e20362c6908d8969#diff-59a07870f8bd3a0ebe35d335d49994c6R176

This is why we need this PR